newIDE/app/src/EventsSheet/EventsTree/Instruction.js

```diff
@@ -120,6 +120,7 @@ class Instruction extends React.Component<Props, *> {
             return <span key={i}>{formattedTexts.getString(i)}</span>;
 
           const parameterType = metadata.getParameter(parameterIndex).getType();
+          console.log(parameterType + ':' + formattedTexts.getString(i));
```

Comment:
Would be cool if you remove all the `console.log` and prettify all your files when you submit a PR. This would reduce the "cognitive overhead" when reviewing PR, and would help me concentrate on the important things ;)

Reply:
sorry about that. I forgot to remove them, because it is still a wip